packages/flutter_test/lib/src/binding.dart

```diff
@@ -985,6 +1039,13 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     _parentZone = null;
     buildOwner!.focusManager.dispose();
 
+    if (TestDefaultBinaryMessengerBinding.instance!.defaultBinaryMessenger
+        .checkMockMessageHandler(
+            SystemChannels.accessibility.name, _announcementHandler)) {
+      _announcementHandler = null;
```

Comment:
Should we also remove the handler from `TestDefaultBinaryMessengerBinding`?

Reply:
Yeah, absolutely. Thanks for the reminder.